CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
 - FIX: provide device type to findConfiguration to achieve a better group match in getEffectiveApiKey (iota-node-lib#1155)
 - FIX: update polling when device is updated by adding endpoint (needs iota-node-lib >= 2.19) (twin for iota-json#602)
+- FIX: Remove preprocess stripping of explicitAttrs (#523)
```

Comment:
I thinks should be better to refer the "parent issue" in the IOTA LIB than a self-reference to the PR number, ie:

```suggestion
- FIX: Remove preprocess stripping of explicitAttrs (iotagent-node-lib#1151)
```